src/actions/sources/select.js

```diff
@@ -119,16 +119,22 @@ export function selectLocation(location: Location, tabIndex: string = "") {
 
     await dispatch(loadSourceText(source));
     const selectedSource = getSelectedSource(getState());
+    if (!selectedSource) {
+      return;
+    }
+
+    const sourceId = selectedSource.get("id");
     if (
       prefs.autoPrettyPrint &&
-      !getPrettySource(getState(), selectedSource.get("id")) &&
+      !getPrettySource(getState()) &&
```

Comment:
is the second argument optional? if not do we need to update getPrettySource definition?

Reply:
nice catch